### ISSUE TITLE
What kind of normalizations, if any, should we be applying to the CXR data?

### ISSUE THREAD

**Issue by geekmdtravis** [geekmdtravis/cs-7643-final-project#14]
Normalizing data before training is considered common to ideal, but it significantly alters the images. Here are three example images with no normalization, normalization to imagenet pre-trained weights,  and dataset specific (greyscale) normalization. Of note, normalization is available as a hyperparam. 

![Image](https://github.com/user-attachments/assets/4ab39e2c-aecb-4cd8-8c6f-6d22b065b84f)

![Image](https://github.com/user-attachments/assets/5f0f17ea-4a13-40b4-b9d7-85383233b3c1)

![Image](https://github.com/user-attachments/assets/99683f35-130d-4244-b1e1-1e4115415d26)
